### ISSUE TITLE
Installer failed to process the preconfiguration file

### ISSUE THREAD

**Issue by pabl-o-ce** [aequitas/terraform-provider-transip#61]
Hi guys,

I'm trying to use Debian or Ubuntu latest versions but don't know why I get this error whenever I try to create with Terraform
![unknown](https://user-images.githubusercontent.com/4999086/190284923-22075bd7-244c-4958-b344-bd575c3562dd.png)

When I use cloud-init without Terraform on Debian latest version work great.

Maybe something I'm doing wrong?

**Comment by JoooostB** [comment]
Hi @pabl-o-ce,

Could you share the Terraform code in which you define this VPS resource?

**Comment by pabl-o-ce** [comment]
Hi @JoooostB 

sure,

this is the repo: https://github.com/pabl-o-ce/crowdsec-tf-workshop-vms

Transip file:
https://github.com/pabl-o-ce/crowdsec-tf-workshop-vms/blob/main/transip_resources.tf

but in there also we have digital ocean and aws.... I can isolate the Transip if you need it :)

**Comment by JoooostB** [comment]
This may be due to `instalFlavour` being set to `installer` by default on [TransIP's API](https://api.transip.nl/rest/docs.html#vps-vps-post). For your use-case this should be `cloudinit` instead. 

@aequitas I think we should add a configuration option to the VPS resource for `instalFlavour`

**Comment by aequitas** [comment]
There was already a issue for that https://github.com/aequitas/terraform-provider-transip/issues/29 should be no problem to add that.

**Comment by JoooostB** [comment]
Indeed, but we may want to migrate to `sdk-v2` first before adding new stuff? Happy to add the feature nonetheless.

**Comment by klausagnoletti** [comment]
Great to hear you guys are working to solve the problem but could you maybe give a time estimate? Currently it's a blocker for things to work properly when I do workshops on CrowdSec (I had an agreement with Richard Koorevaar from TransIP to mention them but I can't since I can't spin a vps up in TransIP because of this..)
Thanks!
